src/lib/accessibility/Behaviors/Menu/MenuItemBehavior.ts

```diff
@@ -27,8 +27,24 @@ export class MenuItemBehavior extends AbstractBehavior<{}, {}>
   }
 
   public generateAriaAttributes(props: any, state: any): object {
+    if (props.submenu) {
+      this.attributes['aria-expanded'] = false
+    }
     return this.attributes
   }
 
-  public changeState(newState: ComponentState): void {}
+  public changeState(newState: ComponentState): void {
```

Comment:
is this called from somewhere? I think it should be enough to add
```
this.attributes['aria-expanded'] = state['submenuOpened']
```
to ``generateAriaAttributes``

Reply:
It is not called yet. We have to consider if we want to re-render React component itself or change attributes independently, how it is done in Focus Zone. As we haven't decided yet, it is just part of the approach introduced buy Sergii.